resources/server/players/player.controller.ts

```diff
@@ -47,7 +47,7 @@ if (!config.general.enableMultiChar) {
     if (resource === GetCurrentResourceName()) {
       const onlinePlayers = getPlayers();
       for (const player of onlinePlayers) {
-        await PlayerService.handleNewPlayerJoined(parseInt(player));
+        await PlayerService.handleNewPlayerJoined(player);
```

Comment:
`getPlayers` still returns a array of strings. Is this done because you have updated types on your end?

Reply:
don't know how this managed to sneak in